src/components/NewsletterForm/NewsletterFormResearchDropDown.tsx

```diff
@@ -15,12 +15,12 @@ export const NewsletterFormResearchDropDown = ({
     <NewsletterFormItem type="dropdown">
       <label
         className="newsletter-form__item-label newsletter-form__item-label--dropdown"
-        htmlFor="dropdown"
+        htmlFor="edit-research-dd"
       >
         Which of these best describes you?
       </label>
       <select
-        id="edit-research-dd"
+        id="edit-research-options"
         name="research_dd"
```

Comment:
if we update the id we may as well do the same with the name i.e. research options makes more sense than research dd